src/accordion/accordion.js

```diff
@@ -27,6 +27,7 @@ export default class Accordion extends React.Component<
       expanded: [],
     },
     renderPanelContent: false,
```

Comment:
Could you add a `TODO` to remove the older `renderPanelContent ` prop in the next major?